### ISSUE TITLE
[Linux] Deb package is 32-bit only?

### ISSUE THREAD

**Issue by terzag** [itchio/itch#425]
Hi,
I'm using Ubuntu Mate 15.10 64-bit. If I try to install the Deb package, it wants to remove core packages from my system, which is apparently triggered from the dependancy on p7zip-full in 32-bit flavor (i.e. I get the same problem if I try to install manually the p7zip-full:i386 package: it wants to remove the archive manager, supposedly to install the 32-bit version of it instead). Is there a reason why you provide only a 32-bit package? Would it be broken if the Deb package was set to "all" architectures rather than "i386" (i.e. would the 32-bit itch.io client be unable to use the 64-bit 7zip)?

**Comment by terzag** [comment]
Nevermind, I just saw in a closed bug report that there is indeed a 64-bit package available on https://github.com/itchio/itch/releases

It should really be listed on the main download page.

**Comment by lahtis** [comment]
Why 32-bit deb version wants to remove playonlinux, p7zip, and asetoneiso packages? Playonlinux using p7zip-full package.

**Comment by fasterthanlime** [comment]
> Why 32-bit deb version wants to remove playonlinux, p7zip, and asetoneiso packages? Playonlinux using p7zip-full package.

is the rest of your distro 32-bit?

**Comment by lahtis** [comment]
yes, i have a 32-bit distro.

**Comment by fasterthanlime** [comment]
need to check that this still happens after #570 is merged

**Comment by fasterthanlime** [comment]
nuts (our update server) now defaults to serving 64-bit packages: .deb if it detects Ubuntu, .rpm if it detects Fedora, .tar.xz (portable) otherwise.

The dependency thing was separate, please open a separate issue if it's still happening.
